src/admin/admin.controllers.system.ts

```diff
@@ -0,0 +1,43 @@
+import { Effect as E, pipe } from "effect";
+import { StatusCodes } from "http-status-codes";
+import type { App } from "#/app";
+import { handleTaggedErrors } from "#/common/handler";
+import { PathsV1 } from "#/common/paths";
+import { paramsValidator, payloadValidator } from "#/common/zod-utils";
+import * as SystemService from "#/system/system.services";
+import {
+  AdminDeleteMaintenanceWindowRequestSchema,
+  AdminSetMaintenanceWindowRequestSchema,
+} from "./admin.types";
+
+export function setMaintenanceWindow(app: App): void {
+  app.put(
```

Comment:
Personally I find the difference between `POST` and `PUT` unclear. Can we/should we stick with `POST` and avoid `PUT` altogether in the system?